src/OpenLoco/src/Ui/Windows/Options.cpp

```diff
@@ -120,11 +120,8 @@ namespace OpenLoco::Ui::Windows::Options
             miscellaneous,
         };
 
-        static void drawTabs(Window* w, Gfx::DrawingContext& drawingCtx)
```

Comment:
how come no other windows need changed?

Reply:
I can detect whats new and old by checking if image is set to ImageIds::tab which is the background and not the content.